app/src/main/java/com/example/ihonboarding/domain/home/model/News.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.ihonboarding.domain.home.model
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "news")
+data class News(
+    @PrimaryKey(autoGenerate = true)
```

Comment:
I think news id comes from the backend so you don't need to make this auto generate

Reply:
okay noted